packages/ui/src/components/molecules/OrderSummary/styles.scss

```diff
@@ -12,6 +12,11 @@
   --fs-order-summary-total-text-size                 : var(--fs-text-size-3);
   --fs-order-summary-total-text-font-weight          : var(--fs-text-weight-bold);
 
+  // Taxes label
+  --fs-order-summary-taxes-label-color        : var(--fs-color-info-text);
+  --fs-order-summary-taxes-text-size          : var(--fs-text-size-tiny);
+  --fs-order-summary-taxes-text-weight        : var(--fs-text-weight-regular);
```

Comment:
should we add the new tokens in the docs?

Reply:
Yes, I discussed this with @renatamottam. There is [a task](https://vtex-dev.atlassian.net/browse/B2BTEAM-1659) to create the documentation after the feature is merged.